karrot/subscriptions/consumers.py

```diff
@@ -57,7 +57,17 @@ def connect(self):
         if 'user' in self.scope:
             user = self.scope['user']
             if not user.is_anonymous:
-                ChannelSubscription.objects.create(user=user, reply_channel=self.channel_name)
+                # scope['client'] may contain client ip address
+                # https://github.com/django/asgiref/blob/e8797d47f4006e4e817fda507ec1eacfed4faf60/specs/www.rst
```

Comment:
I wonder if this includes x-forwarded-for stuff? Might just be local ip (which would just be internal one on the server).

I had a quick look at both daphne and uvicorn and they both seem to support x-forwarded-for for websockets as a far as I could tell. We could check once it's deployed to dev...

Reply:
I checked and it does resolve to the actual client ip, so it seems to include headers. but let's double-check that once deployed to dev!